src/store/journal/journal.reducer.ts

```diff
@@ -50,7 +50,7 @@ export const journalReducer = createReducer(
     payload,
   }): JournalModel => ({
     ...state,
-    // TODO: The reducer has to get the lastRefreshed date from the action
+    // @TODO MES-7134: The reducer has to get the lastRefreshed date from the action
```

Comment:
can probably just delete this it's been there over 2 years and not sure exactly what it means

Reply:
gone